src/Model/AliasesDrushRcEmitter.php

```diff
@@ -34,6 +34,20 @@ public function write(AliasCollection $collection)
         $drushConfigFiltered = implode("\n", array_filter($drushConfig, array($this, 'filterForPantheon')));
         $drushConfigFiltered .= "\n" . '$options["include"][] = drush_server_home() . "/.drush/pantheon/drush8";';
         $drushRCEditor->writeDrushConfig($drushConfigFiltered);
+
+        //copy policy docs 
+
+        $policyFromPath = 'policy/drush8/pantheon_policy.drush.inc';
+        $policyToPath = $this->base_dir . "/pantheon/drush8";
+        if (is_dir($policyToPath)) {
+            rmdir($policyToPath . '/pantheon_policy.drush.inc');
```

Comment:
`rmdir` is for dirs, `unlink` is for files. If you ensure that `copy` uses overwrite then you won't need this step at all.

Reply:
ah oops i used this code because i accidentally made an extra dir with the name of the file but its not supposed to stay in there. thanks for catching it!